sdk/eth-light-client-wasm/README.md

```diff
@@ -0,0 +1,9 @@
+## Build locally
+
+To build the binary, you need to have Rust installed and then the `wasm-pack`. The installation script [can be found here](https://rustwasm.github.io/wasm-pack/).
```

Comment:
```suggestion
To build the binary, you need to have Rust installed and then the `wasm-pack`.   
The installation script [can be found here](https://rustwasm.github.io/wasm-pack/).
```
Could you add two spaces at the end of the line and break it to improve readability for people with screens slightly smaller than yours? :)